src/main/kotlin/org/jetbrains/research/ml/dataset/anonymizer/Anonymizer.kt

```diff
@@ -76,15 +105,16 @@ abstract class Anonymizer(private val tmpDataPath: String) {
         }
     }
 
-    fun anonymizeCsvFile(csvFilePath: String): DataFrame {
+    // TODO: I am not sure that we should use fixture here
```

Comment:
I think we shouldn't use fixtures in main.....

Reply:
I am not sure that I currently know a way how to avoid this...